src/nidcpower/metadata/attributes.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# This file is generated from NI-DCPower API metadata version 20.7.0d409
```

Comment:
this downgrade I assume is accidental

Reply:
It's actually an upgrade from 20.7 to 21.0

Reply:
Yeah, it was. Unlike the other generated .py files, this one didn't have any change, so I shouldn't have included it at all in the PR.

As for the other 3 files, which did have changes, I'm keeping their generated version (21.0.0d38) even though they had a higher version in master.

I assume it's okay for one of the generated files to keep the 20.7 version (since it wasn't affected by this change) while the other 3 generated files to show that they were generated with 20.1, correct?

Reply:
Let's keep the version same across all metadata files, even if it means just the version gets updated.

Reply:
ok, will do.